bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RackawareEnsemblePlacementPolicyImpl.java

```diff
@@ -136,9 +208,11 @@ protected RackawareEnsemblePlacementPolicyImpl initialize(DNSToSwitchMapping dns
                                                               int maxWeightMultiple,
                                                               StatsLogger statsLogger) {
         this.statsLogger = statsLogger;
+        this.bookiesJoinedCounter = statsLogger == null ? null : statsLogger.getOpStatsLogger(BookKeeperServerStats.BOOKIES_JOINED);
```

Comment:
when the statsLogger would be null? we should use NullStatsLogger for the `null` case, rather than passing in a `null`.